opm/polymer/fullyimplicit/FullyImplicitCompressiblePolymerSolver.hpp

```diff
@@ -80,11 +80,23 @@ namespace Opm {
                 : rq(num_phases)
                 , rsSat(ADB::null())
                 , rvSat(ADB::null())
+                , fip()
             {
             }
+
+            enum FipId {
+                FIP_AQUA = BlackoilPropsAdInterface::Water,
+                FIP_LIQUID = BlackoilPropsAdInterface::Oil,
+                FIP_VAPOUR = BlackoilPropsAdInterface::Gas,
+                FIP_DISSOLVED_GAS = 3,
+                FIP_VAPORIZED_OIL = 4,
+                FIP_PV = 5,                    //< Pore volume
+                FIP_WEIGHTED_PRESSURE = 6
+            };
```

Comment:
In my mind possibly worse things than as is, since they are used for indexing into a 7-long vector.

The assumption in the code above is that water/oil/gas is used for indexing elsewhere in the code, and will always be 0/1/2 or a permutation thereof.